README.md

```diff
@@ -72,12 +72,9 @@ ng generate component my-new-component
 ng g component my-new-component # using the alias
 
 # components support relative path generation
-# if in the directory src/app/feature/ and you run
```

Comment:
But they do work, we have tests that verify generation within the folder structure. Because of this I agree with @Meligy that your docs should be in addition to the existing docs.